[2023-10-03T10:10:27.975+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: launches_getter.check_data_in_a_day scheduled__2023-09-28T00:00:00+00:00 [queued]>
[2023-10-03T10:10:28.052+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: launches_getter.check_data_in_a_day scheduled__2023-09-28T00:00:00+00:00 [queued]>
[2023-10-03T10:10:28.053+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 1
[2023-10-03T10:10:28.299+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): check_data_in_a_day> on 2023-09-28 00:00:00+00:00
[2023-10-03T10:10:28.362+0000] {standard_task_runner.py:57} INFO - Started process 3078 to run task
[2023-10-03T10:10:28.391+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'launches_getter', 'check_data_in_a_day', 'scheduled__2023-09-28T00:00:00+00:00', '--job-id', '182', '--raw', '--subdir', 'DAGS_FOLDER/launches.py', '--cfg-path', '/tmp/tmphd3kazzs']
[2023-10-03T10:10:28.392+0000] {standard_task_runner.py:85} INFO - Job 182: Subtask check_data_in_a_day
[2023-10-03T10:10:28.626+0000] {task_command.py:410} INFO - Running <TaskInstance: launches_getter.check_data_in_a_day scheduled__2023-09-28T00:00:00+00:00 [running]> on host f21dc12153d8
[2023-10-03T10:10:29.340+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='launches_getter' AIRFLOW_CTX_TASK_ID='check_data_in_a_day' AIRFLOW_CTX_EXECUTION_DATE='2023-09-28T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-09-28T00:00:00+00:00'
[2023-10-03T10:10:29.431+0000] {taskinstance.py:1454} INFO - No data found on date 2023-09-28
[2023-10-03T10:10:29.483+0000] {taskinstance.py:1373} INFO - Marking task as SKIPPED. dag_id=launches_getter, task_id=check_data_in_a_day, execution_date=20230928T000000, start_date=20231003T101027, end_date=20231003T101029
[2023-10-03T10:10:29.691+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2023-10-03T10:10:29.899+0000] {taskinstance.py:2674} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-03T12:31:47.862+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: launches_getter.check_data_in_a_day scheduled__2023-09-28T00:00:00+00:00 [queued]>
[2023-10-03T12:31:47.992+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: launches_getter.check_data_in_a_day scheduled__2023-09-28T00:00:00+00:00 [queued]>
[2023-10-03T12:31:47.999+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 1
[2023-10-03T12:31:48.155+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): check_data_in_a_day> on 2023-09-28 00:00:00+00:00
[2023-10-03T12:31:48.210+0000] {standard_task_runner.py:57} INFO - Started process 190 to run task
[2023-10-03T12:31:48.265+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'launches_getter', 'check_data_in_a_day', 'scheduled__2023-09-28T00:00:00+00:00', '--job-id', '271', '--raw', '--subdir', 'DAGS_FOLDER/launches.py', '--cfg-path', '/tmp/tmps0gvt_qq']
[2023-10-03T12:31:48.329+0000] {standard_task_runner.py:85} INFO - Job 271: Subtask check_data_in_a_day
[2023-10-03T12:31:48.665+0000] {task_command.py:410} INFO - Running <TaskInstance: launches_getter.check_data_in_a_day scheduled__2023-09-28T00:00:00+00:00 [running]> on host f21dc12153d8
[2023-10-03T12:31:49.112+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='launches_getter' AIRFLOW_CTX_TASK_ID='check_data_in_a_day' AIRFLOW_CTX_EXECUTION_DATE='2023-09-28T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-09-28T00:00:00+00:00'
[2023-10-03T12:31:49.154+0000] {taskinstance.py:1454} INFO - No data found on date 2023-09-28
[2023-10-03T12:31:49.194+0000] {taskinstance.py:1373} INFO - Marking task as SKIPPED. dag_id=launches_getter, task_id=check_data_in_a_day, execution_date=20230928T000000, start_date=20231003T123147, end_date=20231003T123149
[2023-10-03T12:31:49.313+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2023-10-03T12:31:49.585+0000] {taskinstance.py:2674} INFO - 0 downstream tasks scheduled from follow-on schedule check
